common/scylla_cloud_io_setup

```diff
@@ -81,6 +81,172 @@ class aws_io_setup(cloud_io_setup):
             self.disk_properties["read_bandwidth"] = 507338935 * nr_disks
             self.disk_properties["write_iops"] = 57100 * nr_disks
             self.disk_properties["write_bandwidth"] = 483141731 * nr_disks
+        elif idata.instance_class() in ("m5d", "r5d"):
```

Comment:
@syuu1228 it should be `self.idata.instance_class()` - same for following refs ofc